tests/integration/add.js

```diff
@@ -164,6 +164,33 @@ test('adds multiple objects to db', function (t) {
   .catch(t.error)
 })
 
+test('fail validation adding multiple objects to db', function (t) {
+  t.plan(1)
+
+  var name = uniqueName()
+  var store = new Store(name, {
+    PouchDB: PouchDB,
+    remote: 'remote-' + name,
+    validate: function () { throw new Error() }
+  })
+
+  store.add([{
+    foo: 'bar'
+  }, {
+    foo: 'baz'
+  }, {
+    _id: 'foo',
+    foo: 'baz'
+  }])
```

Comment:
I think we can simplify the test here and only create one document.

But it would be interesting to see what happens when we try creating two documents at once, one passes validation and one fails. Could you add a test for that maybe?

Reply:
This test is specifically for multiple additions.
As for your question, my code will add any doc unless all the docs pass validation. Let me know if you'd want that behaviour to change?